pom.xml

```diff
@@ -26,7 +26,7 @@
   (in particular, if you are doing it for the first time), instead do
   'mvn package'.  If you are interested in the full story, see
   https://issues.apache.org/jira/browse/HBASE-6795.
-
+for triggering UTs
```

Comment:
This one makes me confused: you got a successful build before adding this.

Reply:
The pre commit job for HBase is a bit 'intelligent' so it will only run the UTs in the module you touched in PR. But most IPC related UTs are in the hbase-server module, so I have to touch the pom file in the root dir so it will run all the UTs.

The failed UTs are not related. Some UTs in HBase are flaky, it is a long story...

Reply:
And I will revert this commit when merging, no worry.